paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSessionViewModel.kt

```diff
@@ -0,0 +1,48 @@
+package com.stripe.android.customersheet
+
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
+import com.stripe.android.customersheet.injection.CustomerSessionComponent
+import com.stripe.android.customersheet.injection.DaggerCustomerSessionComponent
+
+@OptIn(ExperimentalCustomerSheetApi::class)
+internal class CustomerSessionViewModel(
+    application: Application,
+) : AndroidViewModel(application) {
+
+    fun createSession(
```

Comment:
Can we add a test for the behaviors here? Doesn't have to be everything, but at least changing once instance creates a new component with that instance.